docs/concepts/json_schema.md

```diff
@@ -986,52 +986,6 @@ except ValidationError as e:
     """
 ```
 
-As seen above, annotating a field with a `BaseModel` type can be used to modify or override the generated json schema.
-However, if you want to take advantage of storing metadata via `Annotated`, but you don't want to override the generated JSON
-schema, you can use the following approach with a no-op version of `__get_pydantic_core_schema__` implemented on the
-metadata class:
```

Comment:
This does seem like a terrible example, but is ther ean alternative way to do what it is trying to do?

Reply:
Yep, see https://github.com/pydantic/pydantic/pull/10863#issuecomment-2489536843

You just don't have to use the nasty no-op, so it's even better!